ci/build/build-docker-images.py

```diff
@@ -636,6 +706,11 @@ def push_readmes(merge_build: bool):
         required=True,
         help="Whether to bypass checking if docker is affected",
     )
+    parser.add_argument(
+        "--suffix",
+        required=False,
```

Comment:
we can specify `choices` here to avoid any mistakes

```suggestion
        required=False,
        choices=["aarm64"]
```

Reply:
Good point, I'll add this with a constant. Thanks!